graphql_server/render_graphiql.py

```diff
@@ -1,13 +1,17 @@
 """Based on (express-graphql)[https://github.com/graphql/express-graphql/blob/main/src/renderGraphiQL.ts] and
-(subscriptions-transport-ws)[https://github.com/apollographql/subscriptions-transport-ws]"""
+(graphql-ws)[https://github.com/enisdenjo/graphql-ws]"""
 import json
 import re
 from typing import Any, Dict, Optional, Tuple
 
-from jinja2 import Environment
+try:
+    from jinja2 import Environment
+except ImportError:
+    pass
```

Comment:
This is so that users don't need to install Jinja2 if they use the simple regex renderer. Kinda a dirty hack but should be safe since `Environment` is only imported here for type checking and we already did a runtime type check of `jinja_env`

Reply:
that's fine for this case.

Reply:
This, #101 and #104 are ready for merging now.